delta-app/src/main/java/io/cdap/delta/macros/PropertyEvaluator.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright © 2020 Cask Data, Inc.
+ * Copyright © 2022 Cask Data, Inc.
```

Comment:
nit: copyrights should be start year to current year, i.e. `2020-2022`

Reply:
Thanks, fixed